packages/runtime/src/helpers/types.ts

```diff
@@ -51,3 +51,10 @@ export interface RoutesManifest {
   i18n: I18n
   rewrites: Rewrites
 }
+
+// I have no idea what eslint is up to here but it gives an error
+// eslint-disable-next-line no-shadow
+export const enum ApiRouteType {
+  SCHEDULED = 'experimental-scheduled',
+  BACKGROUND = 'experimental-background',
+}
```

Comment:
This (including comment) was moved from another module due to cyclic imports problem

Reply:
```
// I have no idea what eslint is up to here but it gives an error
// eslint-disable-next-line no-shadow
```

This is needed to be moved as well and not just type - same error happen in current location when you remove comments

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/2483 is the reason - `enum` stuff is problematic with non typescript `no-shadow` rule - I can fix this either here or separately

Reply:
Fixed in https://github.com/netlify/next-runtime/pull/2059/commits/66a9670e413aca7959efcdc65901721981a7ee64